test/tests/multiapps/picard_multilevel/picard_master.i

```diff
@@ -62,12 +62,11 @@
   solve_type = PJFNK
   petsc_options_iname = '-pc_type -pc_hypre_type'
   petsc_options_value = 'hypre boomeramg'
-  picard_max_its = 30
+  fixed_point_max_its = 30
   nl_rel_tol = 1e-8
   nl_abs_tol = 1e-9
-  picard_rel_tol = 1e-8
-  picard_abs_tol = 1e-9
-#  picard_force_norms = true
+  fixed_point_rel_tol = 1e-8
```

Comment:
My big request is that you need to patch apps before this can be merged. That can be painful.

Reply:
This touches the key capability of Griffin and we do not want to see lots of deprecation messages.

Reply:
Yup I'll do that

Reply:
deprecation is actually good idea for this use. Otherwise, it will break many input files that use `picard_max_its` and other old parameters.

We need to give users time to adjust these API changes